tests/integration/__init__.py

```diff
@@ -81,7 +81,7 @@
     # Avoid ${TMPDIR} and gettempdir() on MacOS as they yield a base path too long
     # for unix sockets: ``error: AF_UNIX path too long``
     # Gentoo Portage prefers ebuild tests are rooted in ${TMPDIR}
-    os.environ.get('TMPDIR', tempfile.gettempdir()) if salt.utils.is_darwin() else '/tmp'
+    os.environ.get('TMPDIR', tempfile.gettempdir()) if not salt.utils.is_darwin() else '/tmp'
```

Comment:
This seems like it will change handling for all *other* platforms as well. If I'm a Linux platform, I'd get `/tmp` prior to this change and now I'll get `os.environ.get('TMPDIR', tempfile.gettempdir())`. Would it be better to break this logic up to isolate your change just to Darwin?